src/requirements.txt

```diff
@@ -24,3 +24,4 @@ sympy==1.10.1; python_version >= "3.7" and python_version < "4"
 tqdm==4.64.0; python_version >= "3.6" and python_version < "4" and python_full_version >= "3.7.0"
 typing-extensions==4.3.0; python_version >= "3.7" and python_version < "4" and python_full_version >= "3.7.0"
 urllib3==1.26.11; python_full_version >= "3.7.0" and python_version < "4" and python_version >= "3.7"
+opencv-python==4.5.5.64; python_full_version >= "3.7.0" and python_version < "4" and python_version >= "3.7"
```

Comment:
Have you checked that this version of OpenCV is running smoothly on ARM CPU?

Reply:
What do you mean by running smoothly ? Where I can check that ?

Reply:
OpenCV has some bindings to low level languages, which needs to be compiled (similarly to PyTorch). And since we don't want to build OpenCV for ARM (just like what we did for PyTorch in the early days), we need to make sure that this requirement is installing smoothly on ARM/Raspberry Pi (no error on install & basic command) compared to common CPU architectures